data/Quake Champions

```diff
@@ -0,0 +1,17 @@
+Anarki
+Athena (QC)
+BJ Blazkowicz
+Clutch
+Death Knight
+Doom Slayer
+Eisen
+Galena
+Keel
+Nyx
```

Comment:
Let's add the franchise tag here too, since we'll probably have Nyx from Dota soon-ish.